Tasks/AlexUlanov/Task6/Task6_2.py

```diff
@@ -0,0 +1,14 @@
+a = input("Полидром?\n")
+def palidrome(a):
+    if len(a) % 2 == False:
+        return False
```

Comment:
first of all, the correct way to compare would be len(a) % 2 == 0, your version will work exactly the same, but it's not explicit

secondly, it's possible to have a palindrome with even length, like 'ABBA', but your solution would return False immediately, which is wrong